src/main/ncdio_pio.F90.in

```diff
@@ -1729,13 +1729,9 @@ contains
        do n = 1,ndims_iod
           status = pio_inq_dimlen(ncid,dids(n),dims(n))
        enddo
-#if ({ITYPE}==TYPELOGICAL)
+       status = pio_inq_vartype(ncid, vardesc, xtype)
```

Comment:
Just to double-check: this will return PIO_INT for a logical variable? And will this work with pio1 and pio2?